diff -ru gcc-6.5.0.orig/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc gcc-6.5.0/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc
--- gcc-6.5.0.orig/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc	2018-09-23 18:36:14.000000000 +1000
+++ gcc-6.5.0/libsanitizer/sanitizer_common/sanitizer_platform_limits_posix.cc	2021-08-11 14:07:43.296617564 +1000
@@ -116,6 +116,10 @@
 #  include <asm/ptrace.h>
 #  ifdef __arm__
 typedef struct user_fpregs elf_fpregset_t;
+#   define ARM_VFPREGS_SIZE_ASAN (32 * 8 /*fpregs*/ + 4 /*fpscr*/)
+#   if !defined(ARM_VFPREGS_SIZE)
+#     define ARM_VFPREGS_SIZE ARM_VFPREGS_SIZE_ASAN
+#   endif
 #  endif
 # endif
 # include <semaphore.h>
